### ISSUE TITLE
Bad Nuclei Segmentation performance

### ISSUE THREAD

**Issue by kabbas570** [nibtehaz/MultiResUNet#18]
Dear Authors of MultiResUNet,
Greetings, Thank you for your useful contribution to the research community.

I am re-implementing your model to generate and compare the results with my model for publication in a reputable journal. 
For the other 3 datasets, your model performance is as good as mentioned in the paper, But for this Nuclei Segmentation dataset, the IoU cannot increase beyond 40%.
Dataset link: https://www.kaggle.com/gangadhar/nuclei-segmentation-in-microscope-cell-images

I think you are using the same dataset in your paper, I am using the same loss functions/ learning rates as in your GitHub and trained for epochs ranging from 20 to 80, but still, I could not produce on-par results.
I have to mention the result in my paper, can you please help me with this issue.
Thank You,
Regards,

**Comment by nibtehaz** [comment]
Dear @kabbas570 

Sorry the delayed response, I missed this. 

Thank you for your interest in our project, and I am delighted that you are getting similar result in the 3 datasets.

However, I haven't use the nucleus dataset you linked here, I had used the Murphy lab dataset from here (http://murphylab.web.cmu.edu/) . Your kaggle link says that this dataset is also included but I don't know anything about this sorry.

Nevertheless thanks for sharing the dataset, we will hopefully investigate this in a future work.

Best wishes with your work.
